<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AssignRole_Dialog</class>
 <widget class="QDialog" name="AssignRole_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>180</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="labelUid">
     <property name="text">
      <string>Kart UID: </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="radioPilot">
     <property name="text">
      <string>Pilot</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="radioHostess">
     <property name="text">
      <string>Hostes</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="radioPassenger">
     <property name="text">
      <string>Yolcu</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui> 